recipes/esys-escript/bld.bat

```diff
@@ -0,0 +1,56 @@
+:: Write python configuration, see https://github.com/boostorg/build/issues/194
+@echo using python > user-config.jam
+@echo : %PY_VER% >> user-config.jam
+@echo : %PYTHON:\=\\% >> user-config.jam
+@echo : %PREFIX:\=\\%\\include >> user-config.jam
+@echo : %PREFIX:\=\\%\\libs >> user-config.jam
+@echo ; >> user-config.jam
+xcopy user-config.jam C:\Users\appveyor
+
+:: Start with bootstrap
+call bootstrap.bat
+if errorlevel 1 exit 1
+
+:: Build step
+.\b2 install ^
+    --build-dir=buildboost ^
+    --prefix=%LIBRARY_PREFIX% ^
+    toolset=msvc-%VS_MAJOR%.0 ^
+    address-model=%ARCH% ^
+    variant=release ^
+    threading=multi ^
+    link=static,shared ^
+    --layout=system ^
+    --with-python ^
+    -j%CPU_COUNT%
+if errorlevel 1 exit 1
+
+:: Move dll's to LIBRARY_BIN
+move %LIBRARY_LIB%\boost*.dll "%LIBRARY_BIN%"

```

Comment:
Why build boost here?

Reply:
I've deleted the file. I currently do not have a working windows build of esys-escript.